src/Orbital/README.md

```diff
@@ -0,0 +1,192 @@
+require:
+  - $(this-folder)/../readme.azure.noprofile.md 
+input-file:
+  # - $(repo)/specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-03-01/orbital.json
+  - D:\_Code\azure-rest-api-specs\specification\orbital\resource-manager\Microsoft.Orbital\stable\2022-03-01\orbital.json
+
+module-version: 0.1.0
+title: Orbital
+subject-prefix: $(service-name)
+
+resourcegroup-append: true
+identity-correction-for-post: true
+nested-object-to-string: true
+
+directive:
+  - from: swagger-document 
+    where: $.definitions.SpacecraftsProperties.properties.provisioningState
+    transform: >-
+      return {
+        "type": "string",
+        "readOnly": true,
+        "allOf": [
+          {
+            "$ref": "#/definitions/ProvisioningState"
+          }
+        ],
+        "description": "The current state of the resource's creation, deletion, or modification."
+      }
+  - from: swagger-document 
+    where: $.definitions.ContactsProperties.properties.provisioningState
+    transform: >-
+      return {
+        "type": "string",
+        "readOnly": true,
+        "allOf": [
+          {
+            "$ref": "#/definitions/ProvisioningState"
+          }
+        ],
+        "description": "The current state of the resource's creation, deletion, or modification."
+      }
+  - from: swagger-document 
+    where: $.definitions.ContactProfilesProperties.properties.provisioningState
+    transform: >-
+      return {
+        "type": "string",
+        "readOnly": true,
+        "allOf": [
+          {
+            "$ref": "#/definitions/ProvisioningState"
+          }
+        ],
+        "description": "The current state of the resource's creation, deletion, or modification."
+      }
+  - where:
+      variant: ^Create$|^CreateViaIdentity$|^CreateViaIdentityExpanded$|^Update$|^UpdateViaIdentity$
+    remove: true
+  - where:
+      verb: Set
+    remove: true
+  - where:
+      subject: OperationsResult
+    remove: true
+  - where:
+      subject: Contact
+    set:
+      subject: SpacecraftContact
+  - where:
+      subject: SpacecraftAvailableContact
+    set:
+      subject: AvailableSpacecraftContact
+  - where:
+      subject: ContactProfileTag
+    set:
+      subject: ContactProfile
+  - where:
+      subject: SpacecraftTag
+    set:
+      subject: Spacecraft
+  - where:
+      verb: Update
+      subject: ContactProfile
+      parameter-name: ContactProfileName
+    set:
+      parameter-name: Name
+  - where:
+      verb: Update
+      subject: Spacecraft
+      parameter-name: SpacecraftName
+    set:
+      parameter-name: Name
+  - where:
+      subject: AvailableSpacecraftContact
+      parameter-name: SpacecraftName
+    set:
+      parameter-name: Name
+      alias: SpacecraftName
+  - where:
+      subject: AvailableGroundStation
+      parameter-name: GroundStationName
+    set:
+      parameter-name: Name
+      alias: GroundStationName
+  # Re-name and custom it
```

Comment:
```suggestion
  # Re-name and custom it
```
Customize it for what? could you clarify? The more clear the information is, the reviewer is easier to understand your points.